scalafmt-tests/src/test/resources/rewrite/RedundantBraces.stat

```diff
@@ -427,12 +430,12 @@ class Test extends AnyWordSpec {
 >>>
 class Test extends AnyWordSpec {
   "SUT" when {
-    "statement about test" should {
-      "some test" in {
-        val s = 1
-      }
+      "statement about test" should {
```

Comment:
What is reason if this extra indent?

Reply:
what would you prefer:

```
// option 1: everything is indented consistently (that's the one implemented now)
val b = qual1 op1 {
    1 + 2 + 3 + 4 + 5
  } op2
  (111 * 222 * 333 * 444 * 555) op3 {
    "1 / 2 / 3 / 4 / 5"
  }

// option 2: if first op is block, it's not indented
val b = qual1 op1 {
  1 + 2 + 3 + 4 + 5
} op2
  (111 * 222 * 333 * 444 * 555) op3 {
    "1 / 2 / 3 / 4 / 5"
  }

// option 3: blocks are not indented
val b = qual1 op1 {
  1 + 2 + 3 + 4 + 5
} op2
  (111 * 222 * 333 * 444 * 555) op3 {
  "1 / 2 / 3 / 4 / 5"
}

// option 4: treat one op vs multiple differently
val a = qual1 op1 {
  1 + 2 + 3 + 4 + 5
}
val b = qual1 op1 {
    1 + 2 + 3 + 4 + 5
  } op2
  (111 * 222 * 333 * 444 * 555) op3 {
    "1 / 2 / 3 / 4 / 5"
  }
```

Reply:
@poslegm just a little context: none of these options are ideal, in my opinion, but half of the time spent on this commit was to fiddle with indents, so this is all i can offer :)

* option 1 add special handling for blocks because another rule takes care of them without consideration of the infix formatting; it looks better further down the infix expression but the first indent (frequently, the only) looks unusual -- and caught your attention

* option 2 is inconsistent; plus, if the second infix RHS is also a block, we are back to the same problem as in option 1

* option 3 is what the original infix formatting uses, without any special treatment of blocks and leaving them to the apply rules; *we can stay with that, to reduce the amount of change*

* option 4 is weird as it shifts existing code if one adds another infix op to it

Reply:
ok, i have decided to add another parameter so people can choose between options 1 and 3. the other two will not be available.

Reply:
Sorry for long answer. I think that option 3 is preferable (don't add special indent) because infix operators with block active actively used in testing frameworks and it 4-space indentation looks weird in case of tests

Reply:
ok, done, no unusual indents. `scala-repos` diffs (without `?w=1`) have been updated, uploading rebased code shortly. the trick was to create an indent object that can be disabled, because indent needs to be applied only to the first newline.